reporting.md

```diff
@@ -7,37 +7,43 @@ The reports are generated in the following ways:
 1. [BIRT Reports](reporting_birt.md) for facility level and some MoH HMIS reports 
 2. [Cohort Builder](reporting_cohort_builder.md) for custom data extraction in Excel for more detailed analysis
 
-USING THE COHORT BUILDER
+## USING THE COHORT BUILDER
 
 1.Log into UgandaEMR and select Legacy Systems Administration
 
-![](/assets/cohort_builder_Loginto_UgandaEMR .png)
+![Cohort Builder Login](/assets/cohort_builder_Loginto_UgandaEMR .png)
 
-2.Click on the "Cohort builder" tab in the green navigation ribbon![](/assets/click_on_cohortbuilder.png)
+2.Click on the "Cohort builder" tab in the green navigation ribbon
 
-1. Click on the "Encounter" tab and select "ART\_card summary" and specify the dates.
+![Click Cohort Builder](/assets/click_on_cohortbuilder.png)
 
-![](/assets/click_on_cohortbuilder_tab.png)
+3.Click on the "Encounter" tab and select "ART\_card summary" and specify the dates.
 
-Click on search and view your results.
+![Click Encounter Tab](/assets/click_on_cohortbuilder_tab.png)
 
-1. 1. [UgandaEMR Reports](ugandaemr_reports.md) being developed to replace BIRT and provide a starting point for commonly used cohort builder report exports
+4.Click on search and view your results.
 
-USING THE DATA EXPORT FUNCTION
+*[UgandaEMR Reports](ugandaemr_reports.md) being developed to replace BIRT and provide a starting point for commonly used cohort builder report exports*
 
-1. Click on the "Administration" tab on the green navigation bar.
+## USING THE DATA EXPORT FUNCTION
 
-![](/assets/click_on_administrator_navigation_bar.png)2. Click on "Manage Data Exports" under the Reports section
+1.Click on the "Administration" tab on the green navigation bar.
 
-![](/assets/manage_data_exports.png)
+![Click Addministration Bar](/assets/click_on_administrator_navigation_bar.png)
 
-3. Click on the"Add New Data Export" link under Manage Data Exports
+2.Click on "Manage Data Exports" under the Reports section
 
-![](/assets/add_new_dataexport.png)
+![Manage Data Exports](/assets/manage_data_exports.png)
 
-4. Click on "Define Columns" and ensure that you have "simple column" selected
+3.Click on the"Add New Data Export" link under Manage Data Exports
 
-![](/assets/click_on_define_columns.png)5. Pick the "Encounter Date" value in the "first encounter info section of the drop down list as shown above and rename the Column Name to reflect the "Enrollment Date"
+![Add New DataExport](/assets/add_new_dataexport.png)
 
-6. Save the Data Export and run it.
+4.Click on "Define Columns" and ensure that you have "simple column" selected
+
+![Click Define Columns](/assets/click_on_define_columns.png)
+
+5.Pick the "Encounter Date" value in the "first encounter info section of the drop down list as shown above and rename the Column Name to reflect the "Enrollment Date"
+
+6.Save the Data Export and run it.
```

Comment:
@ODORA0 There need to be setps on how to run the data export, if not add a section Running Data Export then put the text TBD

Reply:
@ssmusoke Added the Running Data Export section with a TBD